Flames/flames_drv.py

```diff
@@ -148,33 +95,73 @@ def shutdown():
         pooferFiringThread.shutdown()
         pooferFiringThread.join()
         pooferFiringThread = None
-    
+
 class PooferFiringThread(Thread):
     TIMEOUT = 1 # 1 second timeout, even if no events
-    
+
     def __init__(self, cmdQueue):
         Thread.__init__(self)
         logger.info("Init Poofer Firing Thread")
         self.cmdQueue = cmdQueue
         self.running = False
+        self.isStopped = False
```

Comment:
I'd either rename this variable or add a comment as to why 'isStopped' is not the inverse of 'running'.